src/cli.ts

```diff
@@ -59,10 +62,93 @@ const noop = (): void => {
     logger.info('noop performed');
 };
 
+
+
+const syncBadPathCounts = async (): Promise<void> => {
+    await useDatabaseTransaction(async () => {
+        const topics = await CourseTopicContent.findAll(
+            {
+                where: {
+                    active: true,
+                },
+                include: [
+                    {
+                        model: CourseWWTopicQuestion,
+                        as: 'questions',
+                        include: [
+                            {
+                                model: CourseQuestionAssessmentInfo,
+                                as: 'courseQuestionAssessmentInfo'
+                            }
+                        ]
+                    }
+                ]
+            }
+        );
+
+        logger.info(`Checking ${topics.length} topics`);
+
+        await topics.asyncForEach(async (topic) => {
+            // For each question in the topic, sum the existence of a path problem.
+            let count = 0;
+            await topic.questions?.asyncForEach(async (question)=>{
+                const {webworkQuestionPath, errors, courseQuestionAssessmentInfo} = question;
+                const additionalProblemPaths = courseQuestionAssessmentInfo?.additionalProblemPaths;
+                const additionalErrors = courseQuestionAssessmentInfo?.errors;
+
+                const isDefinitelyBad = VALID_PG_PATH_REGEX.test(webworkQuestionPath) === false;
+
+                // If this is definitely bad and not already in the error object, add it.
+                if (isDefinitelyBad && (_.isNil(errors) || _.isEmpty(errors) || !(webworkQuestionPath in errors))){
+                    logger.debug(`Updating ${question.id} from topic ${topic.id} with a path error.`);
+                    question.errors = {
+                        [webworkQuestionPath]: [`${webworkQuestionPath} cannot be found.`]
+                    };
+                    question.save();
+                }
+
+                if (topic.topicTypeId === 2) {
```

Comment:
I don't know if we need this check, if it has assessment info since someone went back and forth we might still want those errors

I don't feel too strongly about this

Reply:
We don't want these errors because they'd be displayed on the details page. We don't want to force people to have to clear their assessment paths if they're not using them.